.github/workflows/lint.yml

```
@@ -11,23 +11,20 @@ on:
     strategy:
       fail-fast: true
+    # No more than a few minutes
+    timeout-minutes: 5
 
-    name: Foundry project
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
 
       - name: Set up Node.js
         uses: actions/setup-node@v2
```

actionlint (1.7.4)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)